packages/accounts-controller/src/AccountsController.ts

```diff
@@ -951,6 +965,13 @@ export class AccountsController extends BaseController<
       },
     };
 
+    accountsState[newAccount.id] = newAccountWithUpdatedMetadata;
```

Comment:
Nit: Remove the newline right before this one (so that `const newAccountWithUpdatedMetadata` and this assignment are "associated" together)